src/components/atoms/Button/constants.ts

```diff
@@ -14,7 +14,7 @@ export const variantStyle: { [key in ButtonVariants]: string } = {
 
 export const sizeStyle: { [key in ButtonSizes]: string } = {
     [ButtonSizes.xs]: 'rounded-md px-2.5 py-1',
-    [ButtonSizes.sm]: 'rounded-lg px-3 py-1.5',
+    [ButtonSizes.sm]: 'rounded-lg px-3 py-[7px]',
```

Comment:
Ben had to make the small button 32px in height for the header, and said to increase it in the design system too